qq-literals.cabal

```diff
@@ -18,6 +18,7 @@ library
   build-depends:       base >= 4.7 && < 5,
                        template-haskell >= 2.10.0
   default-language:    Haskell2010
+  ghc-options:         -Werror -Wall -Wincomplete-patterns
```

Comment:
I think it's best to avoid using -Werror in the cabal file, as this means that -Werror will be used even when this library is being built as part of a dependency of something else. I think Hackage prevents you from uploading packages that use -Werror in the cabal file, even.

Reply:
yep, that's fair. Just wanted to demonstrate the problem - hang on, i'll fix the PR.